refinery/config/config.json.erb

```diff
@@ -31,6 +31,7 @@
     }
   },
   "DEFAULT_FROM_EMAIL": "<%= @default_from_email || "webmaster@localhost" %>",
+  "DJANGO_DOCKER_ENGINE_DATA_DIR": "<%= @django_docker_engine_data_dir || "/data" %>",
```

Comment:
Should the default value be `/data/django-docker-engine-data`? Is it even necessary when it's already set in `deployment/manifests/default.pp`?

Reply:
We had to add the default for Travis builds since the `erb` command doesn't populate this setting We're just creating `/data` for Travis builds, but we should probably create the same dir structure across all envs.

Reply:
Yes, there is a configuration drift between environments already. There are now two different ways of setting `DJANGO_DOCKER_ENGINE_DATA_DIR`: explicitly in Puppet for Vagrant and AWS, and implicitly in this template for Travis. This is confusing and could lead to difficult to troubleshoot bugs later. If Puppet were used to configure Travis then there would be no need for workarounds like this.

Reply:
@hackdna Sure, If you wanted to spearhead this that would be great since you are the most knowledgeable abut puppet on the team.